pinot-core/src/main/java/org/apache/pinot/core/segment/index/loader/defaultcolumn/BaseDefaultColumnHandler.java

```diff
@@ -133,6 +133,7 @@ public void updateDefaultColumns()
     // Compute the action needed for each column.
     Map<String, DefaultColumnAction> defaultColumnActionMap = computeDefaultColumnActionMap();
     if (defaultColumnActionMap.isEmpty()) {
+      LOGGER.info("No default column action needed for the segment {}.", _segmentMetadata.getName());
```

Comment:
I would avoid logging this because it will be logged in 99% of the case, where no default columns need to be generated. We already have logs if we do need to generate default columns.

Reply:
updateDefaultColumn(...) does logging but it is buried in the calls. Fine for our needs now. Removed the line.